apps/ledger-live-desktop/src/renderer/families/ethereum/EditTransactionModal/steps/StepFees.tsx

```diff
@@ -0,0 +1,164 @@
+  }
+  return (
+    <Box flow={4}>
+      {mainAccount ? <CurrencyDownStatusAlert currencies={[mainAccount.currency]} /> : null}
+      {account && transaction && mainAccount && (
+        <Fragment key={account.id}>
+          <SendAmountFields
+            account={mainAccount}
+            parentAccount={parentAccount}
+            status={status}
+            transaction={transaction}
+            onChange={onChangeTransaction}
+            bridgePending={bridgePending}
+            updateTransaction={updateTransaction}
+            transactionRaw={transactionRaw}
+          />
+        </Fragment>
+      )}
+      <Alert type="primary">
+        {EIP1559ShouldBeUsed(mainAccount.currency) ? ( // Display the fees info of the pending transaction (network fee, maxPriorityFeePerGas, maxFeePerGas, maxGasPrice)
+          <ul>
+            {t("operation.edit.previousFeesInfo.pendingTransactionFeesInfo")}
+            <li>{`${t("operation.edit.previousFeesInfo.networkfee")} ${feeValue} ${
+              mainAccount.currency.ticker
+            }`}</li>
+            <li>{`${t(
+              "operation.edit.previousFeesInfo.maxPriorityFee",
+            )} ${maxPriorityFeePerGasinGwei} Gwei`}</li>
+            <li>{`${t("operation.edit.previousFeesInfo.maxFee")} ${maxFeePerGasinGwei} Gwei`}</li>
+          </ul>
+        ) : (
+          <ul>
+            {t("operation.edit.previousFeesInfo.pendingTransactionFeesInfo")}
+            <li>{`${t("operation.edit.previousFeesInfo.networkfee")} ${feeValue} ${
+              mainAccount.currency.ticker
+            }`}</li>
+            <li>{`${t("operation.edit.previousFeesInfo.gasPrice")} ${maxGasPriceinGwei} Gwei`}</li>
+          </ul>
+        )}
+      </Alert>
+    </Box>
+  );
+};
+
+export class StepFeesFooter extends PureComponent<StepProps> {
+  state = {
+    transactionHasBeenValidated: false,
+  };
+
+  onNext = async () => {
+    const { transitionTo } = this.props;
+    transitionTo("summary");
+  };
+
+  componentDidMount() {
+    const { account, parentAccount, transaction, transactionHash } = this.props;
+    if (!account || !transaction || !transactionHash) return;
+    const mainAccount = getMainAccount(account, parentAccount);
+    if (mainAccount.currency.family !== "ethereum") return;
+    apiForCurrency(mainAccount.currency)
+      .getTransactionByHash(transactionHash)
+      .then((tx: { confirmations?: number }) => {
+        if (tx.confirmations) {
+          this.setState({ transactionHasBeenValidated: true });
+        }
+      });
+  }
+
+  render() {
+    const { bridgePending, status } = this.props;
+    const { errors } = status;
+    // exclude "NotOwnedNft" and "NotEnoughNftOwned" error if it's a nft speedup operation
+    let errorCount = Object.keys(errors).length;
+    if (
+      errors.amount &&
+      (errors.amount.name.includes("NotOwnedNft") ||
+        errors.amount.name.includes("NotEnoughNftOwned"))
+    ) {
+      errorCount = errorCount - 1;
+    }
```

Comment:
If it doesn't exist already, I think we should create a small helper just to "filter" errors we accept to ignore. Something like:

```typescript
  const filteredErrors = (errors: ValidationIssues, filteredErrors: Error[]): ValidationIssues => {
    const entries = Object.entries(errors);
    const filteredEntries = entries.filter(([, error]) => !filteredErrors.some((FilteredError) => error instanceof FilteredError));

    return Object.fromEntries(filteredEntries)
  }
```

Didn't test this but that's just the idea anyway 🤷‍♂️